docs/openapi.yaml

```diff
@@ -174,11 +180,14 @@ paths:
       - Query
   /request/user_resource_paths:
     post:
-      description: "Return whether the current user has already requested access to\
-        \ the\nspecified resource path(s), including prefixes of the resource path(s).\n\
-        If the previous request was denied or is still in draft status, will\nreturn\
-        \ False.\n\nArgs:\n    resource_paths (list): list of resource paths\n\nReturn:\
```

Comment:
Do we add `Args` and `Return:` as a part of [PXP-8829](https://ctds-planx.atlassian.net/browse/PXP-8829)

Reply:
no
the API swagger documentation is generated automatically from the endpoints docstrings, and we should not document function inputs and outputs in the API docs